cmd/kots/cli/install.go

```diff
@@ -445,10 +450,15 @@ func InstallCmd() *cobra.Command {
 				case storetypes.VersionPendingPreflight:
 					log.ActionWithSpinner("Waiting for preflight checks to complete")
 					if err := ValidatePreflightStatus(deployOptions, authSlug, apiEndpoint); err != nil {
-						log.FinishSpinnerWithError()
-						return errors.Wrap(err, "failed to validate preflight results")
+						log.FinishSpinner()
+						log.Errorf("Preflight checks contain warnings or errors. The application was not deployed")
+						perr := preflightError{}
+						if errors.As(err, &perr) {
+							print.PreflightErrors(log, perr.Results)
+							os.Exit(1)
+						}
+						return err
 					}
-					log.FinishSpinner()
```

Comment:
We should leave this as we need to stop the spinner in the case where preflights pass too.  Otherwise we are left with an indefinite spinner:
```
  • Deploying Admin Console
    • Creating namespace ✓  
    • Waiting for datastore to be ready ✓  
  • Waiting for Admin Console to be ready ✓  
  • Waiting for installation to complete ✓  
  • Waiting for preflight checks to complete ⠇
  • Press Ctrl+C to exit
  • Go to http://localhost:8800 to access the Admin Console

  • Waiting for preflight checks to complete ⠏
```